Makefile.js

```diff
@@ -527,7 +527,7 @@ target.lint = function() {
     }
 
     echo("Validating JavaScript test files");
-    lastReturn = exec(ESLINT + testCache + TEST_FILES);
+    lastReturn = exec(`${ESLINT}${testCache}"tests/**/*.js"`);
```

Comment:
I think `TEST_FILES` refers to the tests that are run by mocha. So changing `TEST_FILES` would cause the files in `conf/` to get run by mocha, which causes runtime errors.